lib/plugins/Label_4A.test.ts

```diff
@@ -42,19 +41,18 @@ test('decodes Label 4A, variant 1', () => {
 
 test('decodes Label 4A, variant 1, no callsign', () => {
   const decoder = new MessageDecoder();
-  const decoderPlugin = new Label_4A(decoder);
 
   // https://app.airframes.io/messages/3452310240
   const text = '101606,1910,.N317FR,,KMDW,----,1,1016,RT0,LT1,';
-  const decodeResult = decoderPlugin.decode({ text: text });
+  const decodeResult = decoder.decode({ label: "4A", text: text });
```

Comment:
this isn't a bad idea, but it's going to make the PR huge. Do we want a separate PR that just makes this change? Or do we want to move the c-band messages to the MessageDecoder suite? I'm leaning towards moving the c-band tests

Reply:
Moving the c band message tests makes sense with two drawbacks:

1. Need to remember to add tests in a second location for types with this prefix
2. More importantly, there's no way to check if the cband prefix accidentally catches non cband messages and incorrectly truncates the message that it passes into a plugin.

Reply:
do we have messages (minus the prefix) that are unique to c-band? If not, i don't think we need to have c-band label-specific tests other than a couple to test the prefix extraction. 

As for the second point, idk. Is it worth slowing down the test suite by making every test run through the plugin selection code just so we can verify this edge case that we don't even know exists?

Reply:
not sure about any cband specific messages, but the regex for the prefix is pretty permissive. i would be surprised if it never matched any non cband messages. maybe it would be better to send more metadata about the source of the message into the decoders and just tell it explicitly if it's cband?

or we could try to decode as cband if the prefix is detected then retry with the prefix still attached if it fails to decode. that wouldn't necessarily catch every edge case though

Reply:
@kevinelliott and @andermatt64 - Do you have any opinions here?